src/views/toggle.js

```diff
@@ -0,0 +1,58 @@
+import View from '../view';
+
+const ENTER_KEY = 13;
+
+export default class ToggleView extends View {
+  get shouldResizeY() {
+    return true;
+  }
+
+  get shouldResizeX() {
+    return true;
+  }
+
+  get isVisible() {
+    return this.component.count > 0;
+  }
+
+  get stickyClass() {
+    return this.component.options.sticky ? 'is-sticky' : 'is-inline';
+  }
+
+  render() {
+    if (!this.iframe) {
+      return;
+    }
+    super.render();
+    this.iframe.parent.setAttribute('tabindex', 0);
+    if (this.component.options.sticky) {
+      this.iframe.addClass('is-sticky');
+    }
+    if (this.isVisible) {
+      this.iframe.addClass('is-active');
+    } else {
+      this.iframe.removeClass('is-active');
+    }
+  }
```

Comment:
is the `this.resize()` remove from the render function for a reason?
it was added here https://github.com/Shopify/buy-button-js/pull/323/files

I actually did wanted to talk to you about if there is a better fix for this as well.